docs/runtimes/job-function.md

```diff
@@ -0,0 +1,35 @@
+(job-function)=
+# Function of type `job`
+
+The `job` type function runs the code in a Kubernetes Pod.
+
+You can create (register) a function with basic attributes such as code, requirements, image, etc. using the 
+{py:meth}`~mlrun.projects.MlrunProject.set_function` method.
+Functions can be created from a single code, notebook file, or have access to the entire project context directory. (By adding the `with_repo=True` flag, the project context is cloned into the function runtime environment.) 
+
+Examples:
+
+
+```python
+# register a (single) python file as a function
+project.set_function('src/data_prep.py', 'data-prep', image='mlrun/mlrun', handler='prep', kind="job")
+
+# register a notebook file as a function, specify custom image and extra requirements 
+project.set_function('src/mynb.ipynb', name='test-function', image="my-org/my-image",
+                      handler="run_test", requirements="requirements.txt", kind="job")
```

Comment:
```suggestion
                      handler="run_test", requirements=["scikit-learn"], kind="job")
```

The `requirements` parameter is problematic (see https://jira.iguazeng.com/browse/ML-3861) - until we straighten things up, I prefer using it this way, at least in examples.